src/dtproject/dtproject.py

```diff
@@ -46,6 +46,7 @@ class Layers:
         self: LayerSelf
         distro: LayerDistro
         base: LayerBase
+        hooks: LayerHooks
```

Comment:
You need a default value here otherwise this becomes required. Always test with `make test-docker` before making a PR. This generates `15 errors` in the tests.

![image](https://github.com/duckietown/lib-dtproject/assets/22665009/dc72f57c-54a4-4b6c-a46c-afda33a9095b)